examples/printerdemo_mcu/zephyr/CMakeLists.txt

```diff
@@ -41,4 +41,4 @@ add_subdirectory(../../.. slint_build)
 target_sources(app PRIVATE src/slint-zephyr.cpp src/main.cpp)
 target_link_libraries(app PRIVATE Slint::Slint)
 
-slint_target_sources(app ../ui/printerdemo.slint)
+slint_target_sources(app ../../printerdemo/ui/printerdemo.slint)
```

Comment:
I find it a bit strange that the demo is in the _mcu folder, but uses the non-mcu version.

Reply:
Done.